ktor-http/ktor-http-cio/jsAndWasmShared/src/io/ktor/http/cio/MultipartJsAndWasm.kt

```diff
@@ -0,0 +1,14 @@
+package io.ktor.http.cio
+
+import io.ktor.utils.io.ByteReadChannel
+import io.ktor.utils.io.cancel
+import io.ktor.utils.io.discard
+import io.ktor.utils.io.exhausted
```

Comment:
Hmm, `ij_kotlin_name_count_to_use_star_import` option from `.editorconfig` is not respected. Are you using an IDE different from IDEA?

```suggestion
import io.ktor.utils.io.*
```